react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,38 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
+const indexes = [0, 1, 2];
+
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
+  renderSquare = index => (
+    <Square key={index} value={this.props.squares[index]} index={index} onClick={this.props.handleClick} />
+  );
+
+  renderRow = rowIndex => (
+    <div key={rowIndex} className={styles.boardRow}>
+      {indexes.map(i => this.renderSquare(rowIndex * 3 + i))}
```

Comment:
If the prop indexes is undefined `.map` break. I'd add `indexes &&`

Reply:
indexes is a constant, sir. You cant say that